src/Auth/TokenAuthenticator.ts

```diff
@@ -101,7 +102,8 @@ export class Class implements Authenticator<UsernameAndPasswordToken> {
 
 			let digestedSchema:ObjectSchema.DigestedObjectSchema = this.context.documents.getSchemaFor( expandedToken );
 
-			this.context.documents.jsonldConverter.compact( expandedToken, token, digestedSchema, this.context.documents );
+			let jsonldConverter:JSONLDConverter = new JSONLDConverter( this.context.getObjectSchema() );
```

Comment:
What was the reason to change the way instances of JSONLDConverter are obtained?